CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.30.0 Release
+- Release Date: April 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+**Note:** v5.9.0 as our Extended Support Release (ESR) is coming to the end of its lifecycle. v5.19.0 will continue to be our current existing ESR until October 15, 2020. [Learn more in our forum post](https://forum.mattermost.org/t/upcoming-extended-support-release-updates/8526).
+
+**Note:** The Channel Moderation Settings feature is supported on mobile app versions v1.30 and later. In earlier versions of the mobile app, users who attempt to post or react to posts without proper permissions will see an error.
+ 
+### Improvements
+ - Improved Mobile performance related to themes and links.
```

Comment:
@amyblais not sure I understand what this means?

Also I feel like it doesn't come across as very impactful when the perf improvements should be the highlight of the release. 

@enahum in v1.30 what would be the highlight perf improvements that made it in?

Reply:
Agree with @esethna Performance was gain by:
* avoiding overdraws.
* tweaks in the post list by adapting the amount of posts that are being rendered per batch (this allows the app to be more responsive)
* Batching all related data when fetching posts
* Enable virtualization in the post list
* Removed unnecessary dimensions and orientations dispatches
* Remove listening for `user is typing` events for channels the user is not viewing
* Remove gif animations on the center channel (gifs will only animate in the gallery view)

Reply:
Another improvement is that we now fetch the statuses of those users that are in the DM list when opening the app and on login.